server.js

```diff
@@ -1,5 +1,14 @@
 require('dotenv').config();
 
+const Sentry = require('@sentry/node');
+const { getEnv } = require('./utils/index');
+
+// Set up Sentry before (almost) everything else, so we can
+// capture any exceptions during startup
+Sentry.init({
+  dsn: getEnv('SENTRY_DSN', ''),
```

Comment:
I added the `''` default so this won't break local development environments, but it'll raise an exception in production if it's not set.